sympy/integrals/tests/test_failing_integrals.py

```diff
@@ -58,22 +55,6 @@ def test_issue_4540():
         ((-sin(1/x) + x*exp(x))*x + x*sin(1/x)), x).has(Integral)
 
 
-@XFAIL
-def test_issue_4551():
-    assert integrate(1/(x*sqrt(1 - x**2)), x).has(Integral)
-
-
-@XFAIL
-def test_issue_4737a():
-    # Implementation of Si()
-    assert integrate(sin(x)/x, x).has(Integral)
-
-
-@XFAIL
-def test_issue_1638b():
-    assert integrate(sin(x)/x, (x, -oo, oo)) == pi/2
```

Comment:
This is not from #1638. There is a similar integral from `0` to `oo` in #4737 which it may have been confused with. The integral above should be `pi` and it is.

Reply:
I think it's this 1638:
https://code.google.com/archive/p/sympy/issues/1638

Looks like that became #4737 when imported into github